.code-samples.meilisearch.yaml

```diff
@@ -0,0 +1,61 @@
+landing_getting_started_1: |-
+  // In app.component.ts
+
+  import { Component } from '@angular/core'
+  import { instantMeiliSearch } from '@meilisearch/instant-meilisearch'
+
+  const searchClient = instantMeiliSearch(
+    "http://localhost:7700",
+    "searchKey"
+  )
+
+  @Component({
+    selector: 'app-root',
+    templateUrl: './app.component.html',
+    styleUrls: ['./app.component.css'],
```

Comment:
Can we omit this style thing?

Reply:
See algolia: https://www.algolia.com/doc/guides/building-search-ui/getting-started/angular/